src/isc-dhcp/patch/0007-If-destination-of-BOOTREQUEST-is-directed-broadcast-.patch

```diff
@@ -80,19 +71,12 @@ index 8e7f632..73eb8a9 100644
  		iface->address_count = 0;
  		iface->address_max = 4;
  	} else if (iface->address_count >= iface->address_max) {
-@@ -863,14 +884,28 @@ add_ipv4_addr_to_interface(struct interface_info *iface,
- 			log_fatal("Out of memory saving IPv4 address "
- 			          "on interface.");
- 		}
--		memcpy(tmp, 
--		       iface->addresses, 
-+		memcpy(tmp,
-+		       iface->addresses,
+@@ -515,9 +534,23 @@ add_ipv4_addr_to_interface(struct interface_info *iface,
  		       iface->address_max * sizeof(struct in_addr));
  		dfree(iface->addresses, MDL);
  		iface->addresses = tmp;
-+
-+		tmp = dmalloc(new_max * sizeof(struct in_addr), MDL);
++		
++                tmp = dmalloc(new_max * sizeof(struct in_addr), MDL);
```

Comment:
ISC-DHCP source uses tabs for indenting. Leading whitespace here should be a single tab to align properly with the rest of the source.

Reply:
Done.